sideboard/tests/__init__.py

```diff
@@ -9,6 +11,26 @@
 from sideboard.lib import config, services
 
 
+def get_available_port():
+    # Binding to port 0 with socket.SO_REUSEADDR will give us an unused port
+    # that we can immediately reuse. This is mostly safe, but on heavily used
+    # systems there is a potential race condition if another process uses
+    # the same port in the time between requesting an available port and
+    # actually using it. This is unlikely, and the worst that will happen is
+    # the tests fail on that particular test run.
+    #
+    # See https://eklitzke.org/binding-on-port-zero
+    #
+    # Ideally we could tell cherrypy to listen on port 0, and then inspect
+    # it to determine what port it's using, but cherrypy doesn't support that
+    # yet (other parts of cherrypy will try to use the port defined in
+    # 'cherrypy.server.socket_port' and end up failing on startup).
```

Comment:
Great comment!  However, as a style note, anytime you have a giant comment right at the beginning of a function, it should probably be a triple-quoted docstring.